test/files/coverageparser.test.ts

```diff
@@ -8,28 +8,26 @@ import { CoverageParser } from "../../src/files/coverageparser";
 suite("CoverageParser Tests", () => {
     teardown(() => sinon.restore());
 
-    const fakeOutputChannel = { appendLine: () => undefined } as unknown as OutputChannel;
+    const fakeOutputChannel = {
+        appendLine: () => undefined,
+    } as unknown as OutputChannel;
 
-    test("filesToSections properly deduplicates coverages @unit", async () => {
-        // Setup a map of test keys and data strings
-        // Note: we include a duplicate key to test deduplication
+    test("filesToSections properly combines coverages @unit", async () => {
         const testFiles = new Map();
-        testFiles.set("/file/123", "datastringhere");
-        testFiles.set("/file/111", "datastring111");
-        testFiles.set("/file/222", "datastring222");
-        testFiles.set("/file/123", "samestringhere");
```

Comment:
As this is a map it only ever has 3 entries, making the test meaningless :(

Reply:
Whoops hah! Darn Ryan from many ages ago 😅.